### PR TITLE
Code cleanups in tests/rpmpgppubkeyfingerprint.c

### PR DESCRIPTION
**Pull request by sshedi**

### CODE REVIEW

File: tests/rpmpgppubkeyfingerprint.c [hunk 1]
```diff
@@ -61,27 +63,32 @@ static int test(struct test *test)
...
 	fprintf(stderr, "pgpPubkeyFingerprint failed on %s\n", filename);
+	fclose(f);
 	return 1;
     }
 
     // We expect success now.
     char *got = rpmhex(fp, fplen);
     if (! got) {
 	fprintf(stderr, "%s: rpmhex failed\n", filename);
+	fclose(f);
```
Comment (ffesti) [inline tests/rpmpgppubkeyfingerprint.c:78]: This should probabaly also free(fp);

File: tests/rpmpgppubkeyfingerprint.c [hunk 2]
```diff
@@ -61,27 +63,32 @@ static int test(struct test *test)
...
 	return 1;
     }
 
     if (!fpr || strcmp(got, fpr) != 0) {
 	fprintf(stderr, "%s:\n     got '%s'\nexpected '%s'\n",
                 filename, got, fpr ? fpr : "<none>");
 	free(got);
+	fclose(f);
```
Comment (ffesti) [inline tests/rpmpgppubkeyfingerprint.c:86]: This should probably also free(fp);

**Comment by pmatilai** [comment]
> I took the liberty to make this whole source look better.

Better is subjective. The indentation was mostly following rpm coding style, now it's not. See Coding style / Indentation in CONTRIBUTING.md. Never, ever make formatting changes at the same time as other changes, and in particular, don't break what wasn't broken. 

It's impossible to review a change like this. In the light of recent events, "small test stuff" certainly needs reviewing too. One thing that did catch my eye despite that is pointless NULL checks before freeing. Don't do that, free() will do it for you anyhow.

In the meanwhile, there *are* real cleanups/simplifications to be made. Like manually calculating string sizes when it could use rasprintf(), and manual getcwd() when we have rpmGetCwd() and malloc() with OOM checking for what's a static buffer that wouldn't need freeing.

**Comment by sshedi** [comment]
Accidentally closed it, I will open a new PR.

**Pull request closed without merging**